zipline/finance/transaction.py

```diff
@@ -36,6 +36,17 @@ def __init__(self, sid, amount, dt, price, order_id, commission=None):
     def __getitem__(self, name):
         return self.__dict__[name]
 
+    def __repr__(self):
+        return """{typename}(order_id={order_id}, sid={sid}, \
+dt={dt}, amount={amount}, price={price})""".format(
+            typename=type(self).__name__,
+            order_id=self.order_id,
```

Comment:
My only immediate reaction to this is that I'm not sure that it's useful to include `order_id` in the repr.  I can't think of a case where I've wanted to see the order id when looking at a transaction.

Also, we probably want a test for this.  People often forget to update a repr when removing or changing a field name, so test coverage is important for them.